src/XAD/Complex.hpp

```diff
@@ -1481,20 +1492,20 @@ XAD_INLINE auto proj(const xad::FReal<T>& x) -> decltype(::xad::detail::proj_imp
 // different expr (derived1, derived2 - returns scalar)
 
 template <class T>
-XAD_INLINE complex<xad::AReal<T>> polar(const xad::AReal<T>& r, const xad::AReal<T>& theta)
+XAD_INLINE complex<xad::AReal<T>> polar(const xad::AReal<T>& r, const xad::AReal<T>& theta = 0)
```

Comment:
good catch! (and below). Just use `xad::AReal<T>()` as default initialiser, instead of `0`, so that it's more generic and [according to the standard](https://en.cppreference.com/w/cpp/numeric/complex/polar).

How did you find out? Which test failed because the initialiser was missing? If there was none, please add a test that fails without the `=0`, and then passes when it's added. We strive to cover all changes by tests.

Reply:
The default initialiser is actually in the C++ standard. It should have been there from the start. Thanks for fixing this.